.gitignore

```diff
@@ -24,6 +24,7 @@ checkstyle.core.xml
 
 # Mac Os cache files
 .DS_store
+.DS_Store
```

Comment:
```suggestion
```

Reply:
My macos creates .DS_Store files

Reply:
Yes but it is already declared above

Reply:
Is it case sensitive?

Reply:
> Is it case sensitive?

Ah nervermind, I didn't notice the difference in case. I think the files are always created using uppercase Store

Reply:
Yep, I'll delete the other cache file then ^^